packages/graphql/src/schema/resolvers/create.ts

```diff
@@ -29,7 +29,7 @@ export default function createResolver({ node }: { node: Node }) {
     async function resolve(_root: any, _args: any, _context: unknown, info: GraphQLResolveInfo) {
         const context = _context as Context;
         context.resolveTree = getNeo4jResolveTree(info);
```

Comment:
As opposed to the solution that you've presented here, you could actually add an `args` argument to `getNeo4jResolveTree` and then augment the args in the resolve tree with the ones from `args`. Then you only need to do this in one place as opposed to doing it in each translate function. 

Happy to walk through this if you're unsure. 🙂

Reply:
That makes sense to me! It was actually my initial approach, but I was hesitant to modify something so deep in the logic. I'll take a stab at that, thanks 😄